pagefaultgames/pokerogue
# [Misc] Eternatus Dialogue reflects # of Classic Sessions Played

## What are the changes?
Before, Eternatus would use the same number of cycles 5643853 in its dialogue. Now, the number of cycles is accurately updated to the player's number of classic sessions played.  

## Why am I doing these changes?
I just thought it would be neat. 

## What did change?
phases.ts - Calculation of number of cycles + determination of any unique suffixes used by English

### Screenshots/Videos
36 classic sessions played --> 5653889
English + Non-Localized Languages (It + Es)
![image](https://github.com/user-attachments/assets/44f1aa16-8ae9-4091-a09f-cf1c8962931e)

German
![image](https://github.com/user-attachments/assets/c6a9f230-b8bf-4d83-9e4c-b92a7216b5e7)

French
![image](https://github.com/user-attachments/assets/1c4cc649-86df-4273-b5c2-4a84cfd377f1)

Portuguese
![image](https://github.com/user-attachments/assets/3a049325-5c1f-4b6c-91c1-07a4002beef3)

Korean
![image](https://github.com/user-attachments/assets/343cdad4-ebff-47e3-a009-ea570f8450bc)

Simplified Chinese 
![image](https://github.com/user-attachments/assets/71d90b91-54c0-413c-83aa-9586f64fe4b5)

Traditional Chinese
![image](https://github.com/user-attachments/assets/6b584440-9166-4ef3-a95f-2d3dc75db659)

## How to test the changes?
Set wave to 200 in overrides.ts and start a new game in classic mode.

Please note that number.toLocaleString() takes the device's language, not the language chosen in the settings. 

## Checklist
